site/content/en/latest/tasks/security/backend-tls.md

```diff
@@ -59,12 +59,35 @@ Store the cert/key in a Secret:
 kubectl create secret tls example-cert --key=www.example.com.key --cert=www.example.com.crt
 ```
 
-Store the CA Cert in another Secret:
+Store the CA Cert in different ways:
+
+{{< tabpane text=true >}}
+{{% tab header="ConfigMap" %}}
 
 ```shell
 kubectl create configmap example-ca --from-file=ca.crt
 ```
 
+{{% /tab %}}
+
+{{% tab header="ClusterTrustBundle" %}}
+
+Save and apply the following resource to your cluster:
+
+```shell
+apiVersion: certificates.k8s.io/v1beta1
+kind: ClusterTrustBundle
+metadata:
+  name: example-ca
+spec:
+  trustBundle: |
+    [content from ca.crt]
```

Comment:
is this valid ? will this command work ?

Reply:
it need users to handle the indentation correctly, I didn't figure out a easy way.